packages/styles/scss/patterns/leadspace/_leadspace.scss

```diff
@@ -153,7 +133,8 @@ $btn-min-width: 26;
         @include carbon--make-col(8, 16);
       }
 
-      .#{$prefix}--leadspace__desc {
+      .#{$prefix}--leadspace__desc,
+      .#{$prefix}--leadspace__ctas {
         @include carbon--make-col-ready;
         @include carbon--make-col(6, 16);
```

Comment:
I think we have to increase the column span for the `ctas` since they are breaking into the second line 
maybe to `carbon--make-col(8,16)`? Can we also try to make the button stack instead of being squished?

https://ibmdotcom-patterns-react-experimental.netlify.com/?path=/story/leadspace--default-with-no-image